src/components/TripCard.js

```diff
@@ -1,9 +1,27 @@
 import styled from 'styled-components';
 
-export default function TripCard({ id, place }) {
+export default function TripCard({ id, place, startDate, endDate }) {
+  const currentStartDate = new Date(startDate);
+  const startYYYY = currentStartDate.getFullYear();
+  let startMM = currentStartDate.getMonth() + 1;
+  let startDD = currentStartDate.getDate();
```

Comment:
Maybe it would be better to rename the variables to "startYear, startMonth, startDay"